src/ir_Mitsubishi.cpp

```diff
@@ -1186,3 +1198,449 @@ String IRMitsubishi136::toString(void) {
+//   nbytes: Nr. of bytes of data in the array. (>=kMitsubishi112StateLength)
+//   repeat: Nr. of times the message is to be repeated.
+//          (Default = kMitsubishi112MinRepeat).
+//
+// Status: ALPHA / Probably working. Needs to be tested against a real device.
+//
+// Ref:
+//   https://github.com/crankyoldgit/IRremoteESP8266/issues/888
+void IRsend::sendMitsubishi112(const unsigned char data[],
+                               const uint16_t nbytes,
+                               const uint16_t repeat) {
+  if (nbytes < kMitsubishi112StateLength)
+    return;  // Not enough bytes to send a proper message.
+
+  sendGeneric(kMitsubishi112HdrMark, kMitsubishi112HdrSpace,
+              kMitsubishi112BitMark, kMitsubishi112OneSpace,
+              kMitsubishi112BitMark, kMitsubishi112ZeroSpace,
+              kMitsubishi112BitMark, kMitsubishi112Gap,
+              data, nbytes, 38, false, repeat, 50);
+}
+#endif  // SEND_MITSUBISHI112
+
+#if DECODE_MITSUBISHI112
+// Decode the supplied Mitsubishi112 message.
+//
+// Args:
+//   results: Ptr to the data to decode and where to store the decode result.
+//   nbits:   Nr. of data bits to expect.
+//   strict:  Flag indicating if we should perform strict matching.
+// Returns:
+//   boolean: True if it can decode it, false if it can't.
+//
+// Status: STABLE / Reported as working.
+//
+// Ref:
+// FIXME
+bool IRrecv::decodeMitsubishi112(decode_results *results, const uint16_t nbits,
+                                 const bool strict) {
+  if (results->rawlen < ((2 * nbits) + kHeader + kFooter - 1)) return false;
+  if (nbits % 8 != 0) return false;  // Not a multiple of an 8 bit byte.
+  if (strict) {  // Do checks to see if it matches the spec.
+    if (nbits != kMitsubishi112Bits) return false;
+  }
+  uint16_t used = matchGeneric(results->rawbuf + kStartOffset, results->state,
+                               results->rawlen - kStartOffset, nbits,
+                               kMitsubishi112HdrMark, kMitsubishi112HdrSpace,
+                               kMitsubishi112BitMark, kMitsubishi112OneSpace,
+                               kMitsubishi112BitMark, kMitsubishi112ZeroSpace,
+                               kMitsubishi112BitMark, kMitsubishi112Gap,
+                               true, _tolerance, 0, false);
+  if (!used) return false;
+  if (strict) {
+    // Header validation: Codes start with 0x23CB26
+    if (results->state[0] != 0x23 || results->state[1] != 0xCB ||
+        results->state[2] != 0x26) return false;
+    if (!IRMitsubishi112::validChecksum(results->state, nbits / 8))
+      return false;
+  }
+  results->decode_type = MITSUBISHI112;
+  results->bits = nbits;
+  return true;
+}
+#endif  // DECODE_MITSUBISHI112
+// Code to emulate Mitsubishi 112bit A/C IR remote control unit.
+//
+// Equipment it seems compatible with:
+//   Brand: Mitsubishi Electric,  Model: PEAD-RP71JAA Ducted A/C
+//   Brand: Mitsubishi Electric,  Model: 001CP T7WE10714 remote
```

Comment:
Oh! Can you please add this to the applicable section at the top of `ir_Mitsubishi.h` too. That's where the list of supported devices is automatically generated from.

Thanks!